config.py

```diff
@@ -40,55 +41,33 @@ class Config:
 settings = Settings()
 
 
-LOGGING_CONFIG = {
-    'version': 1,
-    'disable_existing_loggers': False,
-    'formatters': {
-        'default': {
-            'format': '%(asctime)s - %(name)s - %(levelname)s - %(message)s',
-        },
-    },
-    'handlers': {
-        'routers': {
-            'level': logging.INFO,
-            'class': 'logging.handlers.RotatingFileHandler',
-            'filename': './logs/routers.log',
-            'maxBytes': 1024 * 1024 * 5,  # 5 MB
-            'backupCount': 10,
-            'formatter': 'default',
-        },
-        'bot': {
-            'level': logging.INFO,
-            'class': 'logging.handlers.RotatingFileHandler',
-            'filename': './logs/bot.log',
-            'maxBytes': 1024 * 1024 * 5,  # 5 MB
-            'backupCount': 10,
-            'formatter': 'default',
-        },
-        'main': {
-            'level': logging.INFO,
-            'class': 'logging.handlers.RotatingFileHandler',
-            'filename': './logs/main.log',
-            'maxBytes': 1024 * 1024 * 5,  # 5 MB
-            'backupCount': 10,
-            'formatter': 'default',
-        },
-    },
-    'loggers': {
-        '': {
-            'handlers': ['main'],
-            'level': logging.INFO,
-            'propagate': False,
-        },
-        'routers': {
-            'handlers': ['routers'],
-            'level': logging.INFO,
-            'propagate': False,
-        },
-        'routers.bot.bot': {
-            'handlers': ['bot'],
-            'level': logging.INFO,
-            'propagate': False,
-        },
-    },
-}
+def create_logger(debug_status: bool) -> loguru.logger:
+    """Create project loggers depending on debug status"""
+    if debug_status:
+        logger.add(sink=sys.stderr,
+                   format='{time} | {level} | {exception} {file.path} {function} '
+                          'line:{line} {message}',
+                   enqueue=True,
+                   diagnose=True,
+                   catch=True,
+                   backtrace=True,
+                   filter=lambda record: record['extra'].get('name') == 'debug_logger',
+                   level='DEBUG')
```

Comment:
Не использую такой формат сноса. Это моя просьба. В импортах в том числе.

```suggestion
    logger.add(
        sink=sys.stderr,
        format='{time} | {level} | {exception} {file.path} {function}'
               ' line:{line} {message}',
        enqueue=True,
        diagnose=True,
        catch=True,
        backtrace=True,
        filter=lambda record: record['extra'].get('name') == 'debug_logger',
        level='DEBUG'
    )
```

Пиши так